exampleSite/content/blog/categories/index.md

```diff
@@ -0,0 +1,11 @@
++++
+date = "2018-06-07"
+fragment = "blog"
+weight = 50
+
+#background = ""
+title = "Latest articles"
+
+article_count = 5
+list_category = true
```

Comment:
This was something I wanted to discuss but forgot to mention. I'm not sure if we need to implement a separate fragment to list taxonomies such as categories. I added it here for now so we have a working sample and it would only be a matter of moving the code around and that's done easily. If you think listing taxonomies in the list fragment itself is a good idea I can clean up the code and this file as well.

Reply:
To stay close to what hugo does, it seems that we should use the category to filter the articles and not a specific section.
This adds the complexity, but that each content page for the blog needs a `category = "blog"` or similar, but could make the list fragment cleaner and able to list not just content from a single section, but from across the site.